Framework/Algorithms/test/RebunchTest.h

```diff
@@ -154,46 +154,42 @@ class RebunchTest : public CxxTest::TestSuite {
   Workspace2D_sptr Create1DWorkspaceHist(int size) {
     Workspace2D_sptr retVal(new Workspace2D);
     retVal->initialize(1, size, size - 1);
-    double j = 1.0;
-    for (int i = 0; i < size; i++) {
-      retVal->dataX(0)[i] = j * 0.5;
-      j += 1.5;
-    }
-    j = 1.0;
-    for (int i = 0; i < size - 1; i++) {
-      retVal->dataY(0)[i] = j;
-      retVal->dataE(0)[i] = sqrt(j);
-      j += 1;
-    }
+    BinEdges x(static_cast<size_t>(size), LinearGenerator(0.5, 0.75));
+    Counts y(size - 1);
+    CountVariances e(size - 1);
+
+    std::iota(y.begin(), y.end(), 1.0);
+    std::iota(e.begin(), e.end(), 1.0);
+
+    retVal->setBinEdges(0, x);
+    retVal->setCounts(0, y);
+    retVal->setCountVariances(0, e);
```

Comment:
`retVal->setHistogram(0, x, y);`
Note that you can omit `e`, since it is just the `sqrt`, it will be computed automatically.